include/util/os-compatibility.h

```diff
@@ -0,0 +1,7 @@
+int os_fd_set_cloexec(int fd);
+
+int set_cloexec_or_close(int fd);
+
+int create_tmpfile_cloexec(char *tmpname);
+
+int os_create_anonymous_file(off_t size);
```

Comment:
Add guards to this file

```
#ifndef _WLR_OS_COMPATABILITY_H
#define _WLR_OS_COMPATABILITY_H

// ...

#endif
```

Reply:
shouldn't the guard equals the file name ?

Reply:
Sure, you can call it `_WLR_UTIL_OS_COMPATABILITY_H` if you want.